sw/device/tests/rv_core_ibex_nmi_irq_test.c

```diff
@@ -107,52 +93,59 @@ static void alert_handler_config(void) {
 
   alert_handler_testutils_configure_all(&alert_handler, config,
                                         /*lock=*/kDifToggleDisabled);
-  // Enables alert handler irq.
-  CHECK_DIF_OK(dif_alert_handler_irq_set_enabled(
-      &alert_handler, kDifAlertHandlerIrqClassa, kDifToggleEnabled));
 }
 
 /**
- * Handle the wdog bark nmi irq.
+ * Handle the wdog bark NMI irq.
  */
 static void wdog_handler(void) {
-  top_earlgrey_plic_peripheral_t peripheral;
-  dif_aon_timer_irq_t irq;
-  isr_testutils_aon_timer_isr(plic_ctx, aon_timer_ctx, &peripheral, &irq);
```

Comment:
note that this function here expects the PLIC to be set up. 
I exchanged this with functions that do not make use of the PLIC, but directly clear the IRQ status bit at the peripheral side only.